Sources/Crypto/Key Agreement/DH.swift

```diff
@@ -108,7 +108,11 @@ public struct SharedSecret: ContiguousBytes {
     ///   - outputByteCount: The length in bytes of resulting symmetric key.
     /// - Returns: The derived symmetric key
     public func hkdfDerivedSymmetricKey<H: HashFunction, Salt: DataProtocol, SI: DataProtocol>(using hashFunction: H.Type, salt: Salt, sharedInfo: SI, outputByteCount: Int) -> SymmetricKey {
+        #if os(iOS) && (arch(arm) || arch(i386))
```

Comment:
I'm not sure I follow - if the OS is iOS and the platform is either a phone or the simulator? Can't this be simplified to just exclude all of iOS?